report/2.Analysis/ProblemStatement.tex

```diff
@@ -1,33 +1,45 @@
 \newpage
 \section{Problem definition}
-As a conclusion to the analysis, a list of requirements, a set of test cases that can be used to verify the requirements, and a definitive problem statement can be defined.
-The requirements are goals defined by the group, and are primarily used to determine the success of the project, both in accordance to the learning goals of the semester, as well as in relation to the groups own ambitions.
-
+As a conclusion to the analysis, a list of requirements, a set of test cases that can be used to verify the requirements, and a definitive problem statement will be defined.
+The requirements are goals defined by the group, and are primarily used to determine the success of the project, mainly in relation to the groups own requirements for the project.
+%føler ikke at vores requirements overhovedet er related til studieordningen. For den siger vitterligt bare: embedded device, med MI og/eller RTS.
```

Comment:
kan godt være at vi skal tilføje et requirement eller to der relaterer sig til studieordningen